docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -233,6 +233,16 @@ While this script is running with all hardware connected properly, you should be
 Refer to the `uavcan.tunnel.Broadcast section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#broadcast>`_ in the 
 specification for more details.
 
+.. _arming_status:
+
+uavcan.equipment.safety.ArmingStatus (Data Type ID = 1100)
```

Comment:
Maybe worth adding a note here saying that its a new thing and only on new releases, so people don't expect it to be on things where it isn't